xbmc/cores/VideoPlayer/DVDSubtitles/DVDSubtitleTagMicroDVD.cpp

```diff
@@ -11,13 +11,14 @@
 #include "DVDCodecs/Overlay/DVDOverlayText.h"
 #include "utils/StringUtils.h"
 
-void CDVDSubtitleTagMicroDVD::ConvertLine(CDVDOverlayText* pOverlay, const char* line, int len)
+std::string CDVDSubtitleTagMicroDVD::ConvertLine(const char* line, int len)
```

Comment:
I am pretty sure you can avoid the multiple copies of the string to the callee (specially because those strings can be quite big), unsure how to do it though. Maybe since the `std::string` is created in `ConvertLine` you can use `std::move` to move it to a string argument passed from the callee. @lrusak or @phunkyfish can probably advise better

Reply:
changed in another way